app/src/main/java/com/asfoundation/wallet/billing/vkpay/VkPaymentIABFragment.kt

```diff
@@ -114,11 +114,23 @@ class VkPaymentIABFragment : BasePageViewFragment(),
   override fun onViewCreated(view: View, @Nullable savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
     viewModel.collectStateAndEvents(lifecycle, viewLifecycleOwner.lifecycleScope)
-    bindArguments()
     setupTransactionCompleteAnimation()
+    lifecycleScope.launch {
+      delay(500)  // necessary delay to ensure the superappKit is actually ready.
+      if (SuperappKit.isInitialized()) {
```

Comment:
why do you need a delay since you have this function to verify the initialization?

Reply:
The isInitialized() function from vk sdk should be checking if it is fully initialized, but it's not. So we have no way to check when it's done